invenio_rdm_records/records/jsonschemas/records/record-v4.0.0.json

```diff
@@ -265,18 +265,16 @@
                 "additionalProperties": false,
                 "properties": {
                   "name": {"type": "string"},
-                  "identifier": {"$ref": "local://definitions-v1.0.0.json#/identifier"},
```

Comment:
A newer version (`v5.0.0`) should be created. It can be done after the feature is implemented.